### PR TITLE
Form loading skeleton

### LINKED ISSUE #283: Hubspot form loading animation

### PR DESCRIPTION
**Pull request by samuelopra**
Fix #283 

Test URLs:
- Before: https://main--hubblehomes-com--aemsites.hlx.live/contact-us
- After: https://issue-283--hubblehomes-com--aemsites.hlx.page/contact-us

### CODE REVIEW

File: blocks/embed/embed.js [hunk 1]
```diff
@@ -5,10 +5,95 @@
  */
 import {
   div,
+  span,
 } from '../../scripts/dom-helpers.js';
 import { addFormConfiguration } from '../../scripts/forms-helper.js';
 import { createOptimizedPicture } from '../../scripts/aem.js';
 
+function createLoadingSkeleton() {
```
Comment (codevoyage) [inline blocks/embed/embed.js:13]: Is there any particular reason we chose this skeletal design for the form? It seems more complex and larger than the actual forms on:

	•	https://issue-283--hubblehomes-com--aemsites.hlx.page/contact-us
	•	https://issue-283--hubblehomes-com--aemsites.hlx.page/warranty

File: scripts/hubspot-helper.js [hunk 2]
```diff
@@ -9,7 +52,19 @@ export default function loadHubSpot() {
...
+      if (window.hbspt) {
+        clearInterval(checkHubSpotReady);
+        initializeForms();
+      }
+    }, 100);
+
+    // Set a timeout to prevent infinite checking
+    setTimeout(() => {
+      clearInterval(checkHubSpotReady);
+    }, 10000);
```
Comment (codevoyage) [inline scripts/hubspot-helper.js:68]: Is this 10s going to be enough time in all cases? It might be, but sometimes I don't see the form load.

**Comment by codevoyage** [comment]
@samuelopra  There is a lot of space before the form on the warranty page, please check. 

Screenshot 2024-08-08 at 11 46 33 AM

**Comment by samuelopra** [comment]
> @samuelopra There is a lot of space before the form on the warranty page, please check. I believe it is due to this: ` min-height: 1130px;` Why do we need that?
> 
> https://issue-283--hubblehomes-com--aemsites.hlx.page/warranty
> 
> Screenshot 2024-08-08 at 11 46 33 AM

We need this otherwise it will cause a content shift, increasing CLS as the form loader will expand the container. I'll have to debug this one though

File: blocks/embed/embed.js (hunk 1 above)
Comment (samuelopra) [reply blocks/embed/embed.js:13]: its a very similar size to contact-us. The warranty may be smaller. For the skeleton, it doesn't have to match, just be generic

**Review by varunmitra — APPROVED** [review]

File: blocks/embed/embed.js (hunk 1 above)
Comment (codevoyage) [reply blocks/embed/embed.js:14]: Our forms currently place each field on a separate row, while the placeholder puts two inputs on a single row for the first  To maintain consistency, we can adjust the skeletal form to match the basic structure of having one input field per row, just for cosmetic reasons. 

Screenshot 2024-08-08 at 2 14 34 PM
Screenshot 2024-08-08 at 2 14 49 PM

**Review by codevoyage — APPROVED** [review]

**Pull request merged**